### ISSUE TITLE
WidevineCdm

### ISSUE THREAD

**Issue by olympus123** [iklevente/ChromiumForWindows#12]
Hi, thank you for your great work in this program.
My issue has to do with Widevine.
I want Widevine support in Chromium, that's why I copy in the installlation of chrome the WidevineCdm folder.
Everytime a new version of Chromium gets installed, this folder is not tranfered to the new folder of the new version.
So everytime Chromium gets updated I need to do the same process again and again (copy WidevineCdm folder).
It would be awesome if  ChromiumForWindows could do something about that.
Check the current installation folder before updating and if it has WidevineCdm, copy this folder to a safe place and move it to the new installation folder.
This way people who use Widevine in Chromium could have a complete automated process for updating Chromium with ChromiumForWindows.

**Comment by iklevente** [comment]
Hi! Thanks for the feedback!

I assume you are using the ungoogled version.

I will start working on the new version of the updater which will include woolyss' Chromium API, therefore it will have a completely different (easier) method of checking new Chromium versions and updating them.

I will keep in mind to save the WidevineCdm folder in the future version. :)

**Comment by olympus123** [comment]
> Hi! Thanks for the feedback!
> 
> I assume you are using the ungoogled version.
> 
> I will start working on the new version of the updater which will include woolyss' Chromium API, therefore it will have a completely different (easier) method of checking new Chromium versions and updating them.
> 
> I will keep in mind to save the WidevineCdm folder in the future version. :)

Yes, I am using Marmaduke's ungoogled chromium builds  (ungoogled, widevine, all-codecs+, win64)
Mostly because of all-codecs+, I watch hevc videos in my browser a lot.
Thank you for considering adding support for it, it's a little annoying opening chromium and trying to watch Netflix and suddendly getting unplayable videos, oopps Chromium got updated :)
Thank you again for your work.
